.cargo/config

```diff
@@ -10,3 +10,10 @@ rustflags = [
     # new in 1.60 and generates false positives
     "-Aclippy::only_used_in_recursion",
 ]
+
```

Comment:
Unrelated, but the commit message needs to be reworded to follow the guidelines.

For example, the recommended upper limit for the first line of the commit message is ideally 50-characters, and it should describe the "what", e.g.: Add vector state context switch support.

Reply:
Nit: The commit message looks better now, but can be edited for brevity / clarity. For example, the title can just be "Add vector state context switch support". The remaining lines can be more verbose with something like "This adds changes to allow...". "It also adds...". Note that lines should wrap at 72 characters (don't think we exceeded it, but just noting).